.github/workflows/ci.yaml

```diff
@@ -61,7 +61,17 @@ jobs:
               run: ./tools/yq --help
               working-directory: "${{ steps.scaffold.outputs.dir }}"
               if: "${{ matrix.preset == 'kitchen-sink' }}"
-
+              
+            - name: Python smoke test
+              run: |
+                mkdir app
+                >app/__main__.py echo -e 'import requests\nprint(requests.get("https://api.github.com").text)'
```

Comment:
fyi @illicitonion this is my current "cheap" way of adding some more assertions that developer actions within a newly stamped project work as intended.

Working on a proper scaffolder for new components within the repo.